### PR TITLE
Validate clinical submission test

### PR DESCRIPTION
**Pull request by P-Saha**

### CODE REVIEW

File: utils/gatewayUtils.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const runGqlUpload = async ({ query, variables, jwt, files, asArray = true }) =>
   const formData = new FormData();
   const updatedVariables = { ...variables, files: [] };
   const map = {};
+  updatedVariables.files = [];
```
Comment (hlminh2000) [inline utils/gatewayUtils.js]: this is not needed :) 🔥

File: tests/programs/clinical-submission.js [hunk 2]
```diff
@@ -106,32 +105,7 @@ module.exports = {
...
-  //   browser.assert.visible('#button-clear-selected-file');
-
-  //   browser.click('#button-clear-selected-file').pause(2000);
-
-  //   // Add logic here
-
-  //   browser.expect.element('#button-validate-submission').to.not.have.attribute('disabled');
-  //   browser.expect.element('#button-submission-sign-off').to.have.attribute('disabled');
-  //   browser.expect.element('#button-clear-submission').to.not.have.attribute('disabled');
-  // },
```
Comment (hlminh2000) [inline tests/programs/clinical-submission.js]: this is not needed? can we just take it out?

Comment (P-Saha) [reply tests/programs/clinical-submission.js]: I believe this is already taken out.

Comment (hlminh2000) [reply tests/programs/clinical-submission.js]: oh oops my bad, had a wrong github setting that messed with my reading :P sorry!

**Pull request merged**